plugins/web/opentelemetry-instrumentation-long-task/src/types.ts

```diff
@@ -0,0 +1,20 @@
+import type { SpanAttributes } from "@opentelemetry/api";
+import type { InstrumentationConfig } from "@opentelemetry/instrumentation";
+
+// Currently missing in typescript DOM definitions
+export interface PerformanceLongTaskTiming extends PerformanceEntry {
+  attribution: TaskAttributionTiming[];
+}
+
+export interface TaskAttributionTiming extends PerformanceEntry {
+  containerType: string;
+  containerSrc: string;
+  containerId: string;
+  containerName: string;
+}
+
+export type ObserverCallback = (entry: PerformanceLongTaskTiming) => SpanAttributes;
+
+export interface LongtaskInstrumentationConfig extends InstrumentationConfig {
+  observerCallback?: ObserverCallback;
+}
```

Comment:
Can you please write a small comment to document this callback for potential users?
We usually also add it to the README. you can see examples in other instrumentations such as [aws-sdk](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-aws-sdk#aws-sdk-instrumentation-options)